composer.json

```diff
@@ -21,6 +21,10 @@
   },
   "minimum-stability": "dev",
   "prefer-stable": true,
+  "scripts": {
```

Comment:
* Raise the supported PHP version to the `7.3`.
* Run the following console command in order to generate a psalm report file for the module:
-  `spryker-dev-console dev:analyze-psalm -m SprykerEco.Computop`.
- `spryker-dev-console dev:composer:update-json-files -m SprykerEco.Computop`.

Reply:
Done